augmented_reality/ArApi/ArApi.pri

```diff
@@ -0,0 +1,110 @@
+###############################################################################
```

Comment:
I merged common and api project files and tried to simplified architecture for AR API